src/pages/homepage/hooks/useAuthContext.js

```diff
@@ -0,0 +1,8 @@
+import { useContext } from "react";
+import AuthContext from "../../../context/authContext";
+// custom hook to use the context
+function useAuthContext() {
```

Comment:
please look at the comment i put  in authContext.js file , u dont need to create hook for this u can just export it from the authContext.js file and delete this one